WordPress/Classes/System/WordPressAuthenticatorProtocol.swift

```diff
@@ -8,6 +8,24 @@ protocol WordPressAuthenticatorProtocol {
 
 extension WordPressAuthenticator: WordPressAuthenticatorProtocol {
     static func loginUI() -> UIViewController? {
-        Self.loginUI(showCancel: false, restrictToWPCom: false, onLoginButtonTapped: nil)
+        Self.loginUI(showCancel: false, restrictToWPCom: false, onLoginButtonTapped: nil, continueWithDotCom: { viewController in
+            // TODO: Replce with a remote feature flag.
+            // Enable web-based login for debug builds until the remote feature flag is available.
+            #if DEBUG
+            let webLoginEnabled = true
```

Comment:
If there is no need to update the flag remotely, you can use `extension FeatureFlag: RolloutConfigurableFlag {` to implement the percentage-based rollout locally.

Reply:
In that case, we'd need to release the app to update the rollout percentage, right?

Reply:
Yup

Reply:
Personally, I'd prefer a remote feature flag because we can completely turn it off if there are horrible issues.